src/Struct/FindologicProduct.php

```diff
@@ -984,8 +1000,11 @@ protected function parseCategoryAttributes(
                 $this->navigationCategory
             );
 
+
             if (!Utils::isEmpty($categoryPath)) {
-                $categories = array_merge($categories, [$categoryPath]);
+                if (!in_array($categoryPath, $categories)) {
```

Comment:
Here it already seems to be too late for this. Can this check be done earlier?

Reply:
No because we are only get the category path right before this check. Any where earlier than here, we might not have the correct path that we are building.